### PR TITLE
Added TypeScript Native 0.0.26 alpha

### PR DESCRIPTION
**Pull request by ASDAlexander77**
Added TypeScript Native 0.0.26 alpha

### CODE REVIEW

File: etc/config/typescript.amazon.properties [hunk 1]
```diff
@@ -2,18 +2,18 @@ compilers=&tsnative
 compilerType=typescript
 versionFlag=--version
 
-group.tsnative.compilers=tsc_0_0_20_gc:tsc_0_0_20_nogc
+group.tsnative.compilers=tsc_0_0_26_gc:tsc_0_0_26_nogc
```
[the remark this answers is not in the document]
Comment (ASDAlexander77) [reply etc/config/typescript.amazon.properties:5]: I have added alias. thx

**Review by mattgodbolt — APPROVED** [review]
Thanks!

**Comment by partouf** [comment]
compiler looks to be broken, and it's not clear as to why

**Comment by ASDAlexander77** [comment]
Thank you for your email.

I will have a look, maybe because I have added "alias" incorrectly

I will double check it on local instance

Kind regards
Alex

**Pull request merged**